src/lib/mina_compile_config/mina_compile_config.ml

```diff
@@ -50,3 +50,13 @@ let rpc_heartbeat_send_every_sec = 10.0 (*same as the default*)
 [%%inject "generate_genesis_proof", generate_genesis_proof]
 
 let transaction_expiry_hr = 2
+
+(* limits on Parties.t size *)
+
+let max_proof_parties = 4
```

Comment:
Should these be in runtime config instead of compile config? I think we are trying to move away from compile config more and more, and plan to rip it out eventually.